### PR TITLE
Create bucket before backup (2.4 Only)

### PR DESCRIPTION
**Pull request by itdove**
https://issues.redhat.com/browse/AAP-13119

### CODE REVIEW

File: stories/topics/proc-gcp-backup-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[id="proc-gcp-backup-requirements"]
+
+= Backup requirements
+
+A bucket must be created in order to store the Ansible on Clouds deployment backup.
```
Comment (ianf77) [inline stories/topics/proc-gcp-backup-requirements.adoc:5]: You must create a bucket to store the ...

File: stories/topics/proc-gcp-backup-requirements.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+The bucket will contains the backup information, the secrets and the databases backups.
```
Comment (ianf77) [inline stories/topics/proc-gcp-backup-requirements.adoc:6]: The bucket contains the backup information,

File: stories/topics/proc-gcp-backup-requirements.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+.Procedure
+. Open the Google Cloud console and navigate to the Cloud Storage -> Buckets
```
Comment (ianf77) [inline stories/topics/proc-gcp-backup-requirements.adoc:9]: In the Google Cloud Console, navigate to menu:Cloud Storage[Buckets].

File: stories/topics/proc-gcp-backup-requirements.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
++
```
Comment (ianf77) [inline stories/topics/proc-gcp-backup-requirements.adoc:10]: You don't need the + between lines for this procedure

File: stories/topics/proc-gcp-backup-requirements.adoc [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+. Choose the storage class that fits the most your requirements.
++
+. Choose the control access that fits the most your requirements.
++ 
+. Choose the data protection that fits the most your requirements.
++
+. Click "Create"
+
+.Troubleshooting
+An error could be raised if the bucket name is already used in another project.
```
Comment (ianf77) [inline stories/topics/proc-gcp-backup-requirements.adoc:28]: An error can be raised...

**Review by ianf77 — COMMENTED** [review]
Some comments

File: stories/topics/proc-gcp-backup-requirements.adoc (hunk 1 above)
Comment (itdove) [reply stories/topics/proc-gcp-backup-requirements.adoc:5]: Done

File: stories/topics/proc-gcp-backup-requirements.adoc (hunk 2 above)
Comment (itdove) [reply stories/topics/proc-gcp-backup-requirements.adoc:6]: Done

File: stories/topics/proc-gcp-backup-requirements.adoc (hunk 3 above)
Comment (itdove) [reply stories/topics/proc-gcp-backup-requirements.adoc:9]: Done

File: stories/topics/proc-gcp-backup-requirements.adoc (hunk 4 above)
Comment (itdove) [reply stories/topics/proc-gcp-backup-requirements.adoc:10]: Done

File: stories/topics/proc-gcp-backup-requirements.adoc (hunk 5 above)
Comment (itdove) [reply stories/topics/proc-gcp-backup-requirements.adoc:28]: Done

**Pull request merged**